.gitignore

```diff
@@ -6,3 +6,4 @@
 /node_modules
 /dist
 data/supplementalData.xml
+vendor/
```

Comment:
"vendors" or "vendor"?

Reply:
It is vendor.

Reply:
Fixed.

Reply:
/vendor is better because it ignores only the vendor folder in the root of repo. - the folder we like to ignore. Just like we used the other patterns in same file.

vendor/ ignores a folder with name vendo anywhere in the repo - this may create unwanted issues in future.

Reply:
Thanks. Fixed!